crates/cast/tests/data/scripts/declare/no_contract/Scarb.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "no_contract_script"
+version = "0.1.0"
+
+[dependencies]
+starknet = ">=2.3.0"
+sncast_std = { path = "../../../../../../../sncast_std" }
+
+[lib]
+sierra = true
+casm = true
+
```

Comment:
I guess those are not needed here, works for me locally when removed.
Possibly, the same for other Scarb.toml files in `tests/data/scripts`

Reply:
we can do the cleanup in a separate PR though.

Reply:
removed in newest commit :+1:
left it for map_script, as this one needs it